src/darwin/Framework/CHIPTests/MTRDeviceTests.m

```diff
@@ -2605,6 +2606,166 @@ - (void)test028_TimeZoneAndDST
 #endif // MTR_ENABLE_PROVISIONAL
 }
 
+/**
+ * Given a path relative to the Matter root, create an absolute path to the file.
+ */
+- (NSString *)absolutePathFor:(NSString *)matterRootRelativePath
+{
+    // Find the right absolute path to our file.  PWD should
+    // point to our src/darwin/Framework.
+    NSString * pwd = [[NSProcessInfo processInfo] environment][@"PWD"];
```

Comment:
Doesn't this need to be conditioned on some "not compiling this on iOS" bits?

Reply:
forgot to remove these. not needed anymore. removed.